system/system.proto

```diff
@@ -27,7 +27,7 @@ package gnoi.system;
 import "github.com/openconfig/gnoi/common/common.proto";
 import "github.com/openconfig/gnoi/types/types.proto";
 
-option (types.gnoi_version) = "0.1.0";
+option (types.gnoi_version) = "0.1.1";
```

Comment:
I think this change should increment the minor version .

If there are implementations of gNOI System, we might even want to set to 1.0 per the OpenConfig convention.

Reply:
Agree, lets then follow the convention.